sdk/pr.yml

```diff
@@ -0,0 +1,23 @@
+trigger:
```

Comment:
We will want to update these triggers before merging. Given we plan to name the pipeline "python - pullrequest" I would also suggest renaming this to pullrequest.yml.